TTHAnalysis/python/plotter/susy-sos/sos_plots.py

```diff
@@ -114,15 +121,125 @@ def procs(GO,mylist):
 def sigprocs(GO,mylist):
     return procs(GO,mylist)+' --showIndivSigs --noStackSig'
 
+def createPath(filename):
+    if not os.path.exists(os.path.dirname(filename)):
+        try:
+            os.makedirs(os.path.dirname(filename))
+        except OSError as exc: # Guard against race condition
+            if exc.errno != errno.EEXIST:
+                raise
+
+def prepareSubmitter(name, cmd):
+    template = [l.strip("\n") for l in open("scripts/htcondor_submitter.sh").readlines()]
+    filename="/%s/src//submitJob_%s.sh"%(ODIR,name.replace("_%s"%YEAR,''))
+    if args.allCards:
+        filename=filename.replace("%s"%args.lep,'').replace("_%s"%args.reg,'').replace("_%s"%args.bin,'').replace('__','_')
+    if os.path.isfile(filename): return
+    createPath(filename)
+    f = open(filename, "w")
+    if args.allCards:
+        name=name.replace("_%s"%YEAR,'').replace("%s"%args.lep,'').replace("_%s"%args.reg,'').replace("_%s"%args.bin,'')
+        name=name.replace('__','_')
+        if name.startswith('_'): name=name.replace('_','',1)
+    jobid = 'job_%s'%name
+    for line in template:
+#        line = line.replace("[SCRIPT]"        , "%s/jobs/runJob_%s.sh"%(ODIR,name))
+        line = line.replace("[SCRIPT]"        , "/%s/src//wrapRunners_%s.sh"%(ODIR,name.replace("_%s"%YEAR,'')))
+        line = line.replace("[NAME]"       , name                      )
+        line = line.replace("[DIR]"     , "%s/jobs/"%(ODIR)                       )
+        line = line.replace("[QUEUE]"      , args.queue                                   )
+        f.write(line+"\n")
+    f.close()
+
+def prepareRunner(name, cmd):
+    template = [l.strip("\n") for l in open("scripts/htcondor_runner.sh").readlines()]
+    filename = "%s/jobs/runJob_%s.sh"%(ODIR,name)
+    createPath(filename)
+    f = open(filename, "w")
+    jobid = 'job_%s'%name
+    cmssw  = os.popen("echo $CMSSW_BASE").read().strip('\n')
```

Comment:
Maybe we could use the simpler `os.environ['CMSSW_BASE']` in this line? Or is there a reason not to?